source/features/next-prev-notifications.tsx

```diff
@@ -0,0 +1,26 @@
+import './next-prev-notifications.css';
+import * as pageDetect from 'github-url-detection';
+
+import features from '../feature-manager';
+import observe from '../helpers/selector-observer';
+
+function addNextPreviousBtn(notificationsHeader: Element): void {
+	notificationsHeader.parentElement?.parentElement?.classList.add('flex-items-between');
+	const nextPreviousBtns = document.querySelector('.js-notifications-list-paginator-buttons')?.cloneNode(true);
+	nextPreviousBtns?.classList.add('d-none', 'd-lg-flex');
```

Comment:
Don't use `?.` but just `!.` if you expect the item to be there. We want the code to throw an error if elements have gone missing.